pdns/dnsdistdist/dnsdist-lua-hooks.hh

```diff
@@ -28,8 +28,11 @@ class LuaContext;
 namespace dnsdist::lua::hooks
 {
 using MaintenanceCallback = std::function<void()>;
+using TicketsKeyAddedHook = std::function<void(const char*, size_t)>;
+
 void runMaintenanceHooks(const LuaContext& context);
 void addMaintenanceCallback(const LuaContext& context, MaintenanceCallback callback);
+void setTicketsKeyAddedHook(const LuaContext& context, const TicketsKeyAddedHook& hook);
```

Comment:
Not your doing (mine, actually!), but it looks like `addMaintenanceCallback` and `setTicketsKeyAddedHook` can be made `static` inside `dnsdist-lua-hooks.cc` and thus the `MaintenanceCallback` and `TicketsKeyAddedHook` can become local to that file as well, I think?